### DISCUSSION TITLE
Welcome to nvim-ide Discussions!

### DISCUSSION THREAD

**Discussion by ldelossa** [ldelossa/nvim-ide#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lucholalo33** [comment]
Hello! I'm an engineer in a non-cs role (semi conductor yield improvement). I basically just like to tinker and learn about anything technical. I started using vim a while back when I first got into linux and recently came across nvim. I stumbled across this as I was trying to figure out if I could use nvim as an IDE to keep learning/practicing python. This is pretty damn cool plugin, and so far I've had a fun time setting it up and figuring it out.
